dbt/include/bigquery/macros/adapters/apply_grants.sql

```diff
@@ -11,10 +11,86 @@
 {% endmacro %}
 
 
-{%- macro bigquery__get_grant_sql(relation, privilege, grantee) -%}
+{%- macro bigquery__get_grant_sql(relation, privilege, grantees) -%}
+    {{ bigquery__get_grant_sql__option_3(relation, privilege, grantees) }}
+{%- endmacro -%}
+
+
+{%- macro bigquery__get_revoke_sql(relation, privilege, grantees) -%}
+    {{ bigquery__get_revoke_sql__option_3(relation, privilege, grantees) }}
+{%- endmacro -%}
```

Comment:
Quickly swap between options by changing the macro that gets called (`*__option_{x}`).

Test via:
`python -m pytest tests/functional/adapter/test_grants.py`